Dockerfile

```diff
@@ -83,3 +83,5 @@ ONBUILD RUN rm -rf /var/www/html && ln -s /app/public /var/www/html
 
 EXPOSE 80
 CMD ["/bin/bash", "/run.sh"]
+RUN useradd -d /home/username -m -s /bin/bash behance
```

Comment:
@anshupande I would not leave any references to behance in this container. how can we make the user a bit more generic?